tests/test_scheduling.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.conf import settings
+from atmo.aws import s3
+from atmo.scheduling import get_spark_job_results
+
+
+def test_get_spark_job_results_empty(monkeypatch):
+    def mocked_list_objects(*args, **kwargs):
+        return {}
+    monkeypatch.setattr(s3, 'list_objects_v2', mocked_list_objects)
```

Comment:
I think using the monkeypatch fixture is okay, but in case we want to do more complex checks, using the mocker fixture which uses mock allows testing if the method was called, resetting, etc. Just for future reference that monkeypatch is limited in its scope